modules/cnv-cdrom-wizard-fields-web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-create-vms.adoc
+// * cnv/cnv_vm_templates/cnv-creating-vm-template.adoc
+
+[id="cnv-cdrom-wizard-fields-web_{context}"]
+= CD-ROM fields
+
+|===
+|Source | Description
+
+|Container
+|Specify the container path. For example: kubevirt/fedora-registry-disk: latest.
```

Comment:
if `kubevirt/fedora-registry-disk: latest` is something that the user types in, please put it in backticks ``

Reply:
Added the backticks to format the entry path example.